build.zig

```diff
@@ -17,7 +17,7 @@ pub fn build(b: *std.Build) !void {
 
     const config_header = b.addConfigHeader(
         .{
-            .style = .{ .cmake = b.path("src/config-cmake.h.in") },
+            .style = .{ .cmake = b.path("config-cmake.h.in") },
```

Comment:
There are a couple of regressions, here and below "pcre2_compile_cgroup.c".

Reply:
oh! i see what you mean now. took me a while to understand this (namely figuring out that I wasn't on master!)

Reply:
Note that we won't target PRs at old release branches like 10.45. Aiming for master is correct, since the next release will come from there.